doc/rst/meta/modules/packages/Sort.rst

```diff
@@ -0,0 +1,11 @@
+.. TODO: remove this stub file
+
+.. default-domain:: chpl
+
+.. module:: Sort
+   :synopsis: Supports standard algorithms for sorting data.
+
+Sort
+====
+
+The Sort module documentation has been moved to the :ref:`standard library documentation <modules>`.
```

Comment:
Can we link to the new page for the Sort module docs instead of the page for all modules? It would be one less click for someone trying to find the documentation for Sort and makes more sense IMO.

Reply:
I originally tried that, but because there are now 2 `Sort` modules according to sphinx, the links get garbled. I agree it makes for a better user experience, let me try a few more things

Reply:
It's not a requirement that we do this. If it's too much hassle or requires a terribly hacky solution, I'm okay with leaving it as is.
You might have tried this already, but first reaction was to create a placeholder `.. _sort:` at the top of the page in `Sort.chpl` for Sphinx to pick up . It pollutes this specific link just a little bit, but overall is okay IMO.

Reply:
Putting `.. _sort:` in `Sort.chpl` doesn't quite work since `chpldoc` inserts the module header above it, which results in the link being halfway down the page. However, I used `modules/internal/fixInternalDocs.sh` to insert this as a post-process step, which does work